examples/risk_and_security/retail_agent/README.md

```
@@ -544,6 +544,20 @@ middleware:
 
 See `configs/config-with-defenses.yml` for a working example with multiple defense layers at both function and workflow levels.
 
+> In this example, using the attached `config-with-defenses.yml`—where all enabled defenses operate in **redirection** mode—and re-running the same red teaming scenarios resulted in the overall attack success score dropping from **0.7 (baseline, no defenses)** to **0.0**. This illustrates how the provided defense configuration fully mitigates the demonstrated attacks while allowing the retail agent workflow to continue operating normally. 
+
+<table>
+  <tr>
+    <td align="center">
+      <b>Before Defenses</b><br/>
+      <img src="../../../docs/source/_static/attack-score.png"/>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)